### ISSUE TITLE
Seeting for Google+, microformats Schema.org duplicate when there is a rerender

### ISSUE THREAD

**Issue by definder** [s-yadav/react-meta-tags#18]
When there is first render we have:
```
<title>Sites's title</title>
<meta itemprop="name" content="Site's name"/>
```
But when there is second render we have (first meta tag did'n remove):
```
<meta itemprop="name" content="Site's name"/>
<title>Sites's title 2</title>
<meta itemprop="name" content="Site's name 2"/>
```

My render function:
```
render(){
   const {title} = this.props;
    return (
        <MetaTags>
            {!!title && (
                <React.Fragment>
                    <title>{title}</title>
                    <meta itemProp="name" content={title}/>
                </React.Fragment>
            )}
        </MetaTags>
    );
}
```

**Comment by s-yadav** [comment]
Because itemprop is not used to find duplicates. Will check this out.

**Comment by s-yadav** [comment]
Fixed in v0.5.0. Let me know if issue still persist.
